server/src/main/java/io/druid/server/metrics/HistoricalMetricsMonitor.java

```diff
@@ -53,29 +54,42 @@ public boolean doMonitor(ServiceEmitter emitter)
-                                                                                 .setUser2(serverConfig.getTier());
+      final ServiceMetricEvent.Builder builder =
+          new ServiceMetricEvent.Builder().setDimension(DruidMetrics.DATASOURCE, dataSource)
+                                          .setDimension("tier", serverConfig.getTier())
+                                          .setDimension("priority", String.valueOf(serverConfig.getPriority()));
 
-      emitter.emit(builder.build("server/segment/used", used));
+
+      emitter.emit(builder.build("segment/used", used));
       final double usedPercent = serverConfig.getMaxSize() == 0 ? 0 : used / (double) serverConfig.getMaxSize();
-      emitter.emit(builder.build("server/segment/usedPercent", usedPercent));
+      emitter.emit(builder.build("segment/usedPercent", usedPercent));
     }
 
     for (Map.Entry<String, Long> entry : serverManager.getDataSourceCounts().entrySet()) {
       String dataSource = entry.getKey();
       long count = entry.getValue();
       totalCount += count;
-      final ServiceMetricEvent.Builder builder = new ServiceMetricEvent.Builder().setUser1(dataSource)
-                                                                                 .setUser2(serverConfig.getTier());
+      final ServiceMetricEvent.Builder builder =
+          new ServiceMetricEvent.Builder().setDimension(DruidMetrics.DATASOURCE, dataSource)
+                                          .setDimension("tier", serverConfig.getTier())
+                                          .setDimension(
+                                              "priority",
+                                              String.valueOf(serverConfig.getPriority())
+                                          );
 
-      emitter.emit(builder.build("server/segment/count", count));
+      emitter.emit(builder.build("segment/count", count));
     }
 
-    final ServiceMetricEvent.Builder builder = new ServiceMetricEvent.Builder().setUser2(serverConfig.getTier());
-    emitter.emit(builder.build("server/segment/totalUsed", totalUsed));
+    final ServiceMetricEvent.Builder builder =
+        new ServiceMetricEvent.Builder().setDimension("tier", serverConfig.getTier())
+                                        .setDimension(
+                                            "priority",
+                                            String.valueOf(serverConfig.getPriority())
+                                        );
+    emitter.emit(builder.build("segment/totalUsed", totalUsed));
     final double totalUsedPercent = serverConfig.getMaxSize() == 0 ? 0 : totalUsed / (double) serverConfig.getMaxSize();
-    emitter.emit(builder.build("server/segment/totalUsedPercent", totalUsedPercent));
-    emitter.emit(builder.build("server/segment/totalCount", totalCount));
+    emitter.emit(builder.build("segment/totalUsedPercent", totalUsedPercent));
+    emitter.emit(builder.build("segment/totalCount", totalCount));
```

Comment:
ok, it's up to you, since you came up with the conventions, but I like things being consistent.